gcc/po/gcc.pot

```diff
@@ -1288,7 +1288,7 @@ msgstr ""
 #, no-c-format
 msgid ""
 "-frust-compile-until=[ast|attributecheck|expansion|nameresolution|lowering|"
-"typecheck|privacy|unsafety|const|copimlation|end]             When to stop "
+"typecheck|privacy|unsafety|const|borrowcheck|copimlation|end]             When to stop "
```

Comment:
Hmm, it is generated by gettext but tracked by git. So it should be a part of the PR, but not manually changed. Do you know how?

Reply:
I did change lang.opt as well.

Reply:
ah nice. I hadn't got to that part of the PR yet. I think that file is generated by GCC on a nightly basis or something, so it shouldn't be part of the PR - in fact, it still contains the `comiplation` typo or whatever, which was fixed in lang.opt. It will be fixed and re-generated once we do a full merge upstream I assume

Reply:
From looking at the git log, I don't feel it is automatic. But I don't know.

Reply:
Regeneration of `gcc/po/gcc.pot` is semi-automatic: it's a (scripted) manual process.  It's correct that changes to `[...].pot` files are not to be included in general commits.  (They won't do any harm, though; these files will be regenerated before being sent to the translation teams.)

Reply:
Removed from PR.